src/groupeddataframe/show.jl

```diff
@@ -6,31 +6,51 @@ function Base.show(io::IO, gd::GroupedDataFrame;
                    rowlabel::Symbol = :Row,
                    summary::Bool = true)
     N = length(gd)
-    keys = join(':' .* string.(names(gd.parent)[gd.cols]), ", ")
+    keynames = names(gd.parent)[gd.cols]
+    parent_names = names(gd.parent)
+    keys = join(':' .* string.(keynames), ", ")
+
     keystr = length(gd.cols) > 1 ? "keys" : "key"
     groupstr = N > 1 ? "groups" : "group"
     summary && print(io, "$(typeof(gd)) with $N $groupstr based on $keystr: $keys")
     if allgroups
         for i = 1:N
             nrows = size(gd[i], 1)
             rows = nrows > 1 ? "rows" : "row"
-            print(io, "\nGroup $i: $nrows $rows")
+
+            identified_groups = [':' * string(parent_names[col], " = ", first(gd[i][col])) 
```

Comment:
I've just realized that when the grouping column is a string or symbol, it's printed as e.g. `:x = a` rather than `:x = "a"` or `:x = :a`. I guess we should change this (using `repr` or `ourshowcompact`)?

Reply:
https://github.com/JuliaData/DataFrames.jl/pull/1684